integrations/ultralytics-yolov3/README.md

```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+# SparseML Ultralytics YOLOv3 Integration
+
+This directory combines the SparseML recipe-driven approach with the 
+[ultralytics/yolov3](https://github.com/ultralytics/yolov3) repository.
+By integrating the robust training flows in the `yolov3` repository with the SparseML code base,
+we enable model sparsification techniques on the popular [YOLOv3 architecture](https://arxiv.org/abs/1804.02767)
+creating smaller and faster deployable versions.
+The techniques include, but are not limited to:
+- Pruning
+- Quantization
+- Pruning + Quantization
+- Sparse Transfer Learning
+
+## Highlights
+
+- Blog: [YOLOv3 on CPUs: Sparsifying to Achieve GPU-Level Performance](https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/)
+- Inference Example: [DeepSparse YOLOv3 Example](https://github.com/neuralmagic/deepsparse)
+
+## Tutorials
+
+- Coming soon!
+
+## Installation
+
+To begin using this integration, run the following command in the root directory:
```

Comment:
@markurtz and @bfineran is it obvious in the root directory of [what]? sparseml?

Reply:
root directory of the integration, rewording to put integration at the end